contrib/scripts/protocgen.sh

```diff
@@ -30,5 +30,5 @@ for dir in $proto_dirs; do
     $(find "${dir}" -maxdepth 1 -name '*.proto')
 done
 
-cp -r ./github.com/umee-network/umee/* ./
+cp -r ./github.com/umee-network/umee/v*/x/* x/
```

Comment:
Moves `make proto-gen` output, such as `./v2/x/leverage/types/genesis.pb.go`, to the intended output folder without `v2`.

The previous line was doing something similar for non-v2 output.